DigitalLearningSolutions.Web/Views/LearningMenu/Tutorial/_TutorialNextLink.cshtml

```diff
@@ -0,0 +1,56 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.LearningMenu
+@model TutorialNextLinkViewModel
+
+@if (Model.NextTutorialId != null)
+{
+  <a class="nhsuk-back-link__link next-link"
+     asp-controller="LearningMenu"
+     asp-action="Tutorial"
+     asp-route-customisationId="@Model.CustomisationId"
+     asp-route-sectionId="@Model.SectionId"
+     asp-route-tutorialId="@Model.NextTutorialId">
+
+    Go to next tutorial
+    <svg class="nhsuk-icon nhsuk-icon__chevron-left" id="next-arrow" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true">
+      <path d="M15.5 12a1 1 0 0 1-.29.71l-5 5a1 1 0 0 1-1.42-1.42l4.3-4.29-4.3-4.29a1 1 0 0 1 1.42-1.42l5 5a1 1 0 0 1 .29.71z"></path>
+    </svg>
+  </a>
+}
+else if (Model.PostLearningAssessmentPath != null)
+{
+  <a class="nhsuk-back-link__link next-link"
+     href="">
+
+    Go to post learning assessment

```

Comment:
Could you add a comment on the post learning assessment ticket to make sure this link is updated and tested as part of that ticket? 🙂

Reply:
Which ticket do you mean? I guess HEEDLS-189 but I'm not sure 🙂

Reply:
Yup, that one please 🙂

Reply:
Done 🙂